src/styles/bloks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: center;
+
+  &__item {
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
```

Comment:
Ensure that the path to the `star.svg` image is correct and that the image exists at the specified location. This is crucial for the background image to display properly.